### DISCUSSION TITLE
Viewing Groups/Whitelists/Blacklists for ACL Plugin

### DISCUSSION THREAD

**Discussion by vcastane** [Kong/kong#7231] [Ideas and feature requests]
It seems the right now there is no way to get a list of the groups you have created and their associated permissions. Or get a list of all consumers within each group.

These would be incredibly useful tools for me and other developers.

Same goes for whitelist/blacklist and way to see global ACL information.

Thanks!

**Comment by subnetmarco** [comment]
This may also be related to #505 and would be fixed by [this proposal](https://github.com/Mashape/kong/issues/505#issuecomment-176495650).

**Comment by vcastane** [comment]
Awesome! This is exactly what I was thinking.

Is there a working timeline for this?

**Comment by subnetmarco** [comment]
Another solution would be to extend the ACL plugin to support groups in a better way, as opposed to #505 which implements them in core. But implementing them on core would allow us to fix the structural inconsistency where you can have API-specific plugins, but not API-specific consumers.

Feel free to join the conversation. We didn't make any definitive decision yet, so the implementation will start once we all agree.
